diagnostic.md

```diff
@@ -45,9 +65,18 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big {}
 <!-- your answer ends here -->
+#dontent-pane {
+}
+
+.important .red {
```

Comment:
This will select elements that have ```red``` as a class, that is inside of an element that has ```important``` as a class.  To select elements that have both ```important``` and ```red```, use ```.important.red```.